### PR TITLE
Expose signature value in JWS signatures

### PR DESCRIPTION
**Pull request by bifurcation**
Boulder needs to do calculations on the signature value in a JWS.  This PR just changes `signature` to `Signature` so that it's visible.

### CODE REVIEW

**Comment by csstaub** [comment]
Hi @bifurcation, thanks for the pull request! But before we can merge it you will have to sign our [Individual Contributor License Agreement](https://docs.google.com/a/squareup.com/forms/d/13WR8m5uZ2nAkJH41k7GdVBXAAbzDk00vxtEYjd6Imzg/viewform?ndplr=1&formkey=dDViT2xzUHAwRkI3X3k5Z0lQM091OGc6MQ). Unfortunately we can't accept contributions without it. Make sure to add your GitHub username so I can cross-reference it to your account. If you have concerns about the agreement feel free to contact me privately (see profile for contact info).

File: jws.go [hunk 1]
```diff
@@ -47,9 +47,9 @@ type JsonWebSignature struct {
 // Signature represents a single signature over the JWS payload and protected header.
 type Signature struct {
 	Header    JoseHeader
+	Signature []byte
```
Comment (csstaub) [inline jws.go]: Let's add documentation for these fields

**Comment by bifurcation** [comment]
@cstaub You should have me signed up to the ICLA, I've updated this branch with the new master, and I've added some comments.  Ready to go once tests pass?

**Comment by csstaub** [comment]
Looks good to me. Thanks!

**Pull request merged**